src/Slices/ServiceInventory/UI/ServiceInventory.tsx

```diff
@@ -43,6 +42,31 @@ const PreppedServiceInventory: React.FC<{ service: ServiceModel }> = ({
   />
 );
 
+interface ServiceInventoryContextProps {
```

Comment:
any objections to put the labels into an object? Feels like it make sense to group them to me.

Reply:
grouped